src/arrayMethodReduce.js

```diff
@@ -4,8 +4,20 @@
  * Implement method Reduce
  */
 function applyCustomReduce() {
-  [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+  [].__proto__.reduce2 = function(accumulator, ...startValue) {
```

Comment:
```suggestion
  [].__proto__.reduce2 = function(accumulator, initialValue) {
```

Reply:
Why we can't use the ...args to handle undefined as a value?

Reply:
Because the `initialValue` is one, there cannot be 3 or 4 of them. BTW: if you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review
Fix your code by recommendation